Formula/sqsh.rb

```diff
@@ -0,0 +1,102 @@
+class Sqsh < Formula
+  desc "Sybase Shell"
+  homepage "https://sourceforge.net/projects/sqsh/"
+  url "https://downloads.sourceforge.net/project/sqsh/sqsh/sqsh-2.5/sqsh-2.5.16.1.tgz"
+  sha256 "d6641f365ace60225fc0fa48f82b9dbed77a4e506a0e497eb6889e096b8320f2"
+
+  deprecated_option "enable-x" => "with-x11"
+
+  depends_on :x11 => :optional
+  depends_on "freetds"
+  depends_on "readline"
+
+  # this patch fixes detection of freetds being installed, it was reported
+  # upstream via email and should be fixed in the next release
+  patch :DATA
```

Comment:
What date and what email? I'm tempted to say we wait for a new release before we re-add this.